### PR TITLE
kubelet/rkt: add container/image gc for rkt.

### PR DESCRIPTION
**Pull request by yifan-gu**
Not sure why the rkt container gc is disabled.
This PR reenables it. It also adds image gc.

/cc @dchen1107 @yujuhong

### CODE REVIEW

**Comment by dchen1107** [comment]
ok to test

File: pkg/kubelet/rkt/rkt.go [hunk 1]
```diff
@@ -72,17 +72,17 @@ const (
 	// the grace period from MinAge in ContainerGCPolicy.
 	//
 	// Duration to wait before discarding inactive pods from garbage
-	defaultGracePeriod = "1m"
+	defaultGracePeriod = "0"
 	// Duration to wait before expiring prepared pods.
-	defaultExpirePrepared = "1m"
+	defaultExpirePrepared = "0"
```
Comment (yujuhong) [inline pkg/kubelet/rkt/rkt.go]: Why changing the periods to 0?

Comment (yifan-gu) [inline pkg/kubelet/rkt/rkt.go]: @yujuhong Just for testing, I changed so I can see the result soonner. Changed back.
btw, we should derive this period from gc policy's MinAge, but that requires the gc definition stuff to be moved to kubelet/container or other packages, not in the kubelet.

Comment (yujuhong) [inline pkg/kubelet/rkt/rkt.go]: Moving the gc policy out kubelet sounds good to me.

**Comment by yujuhong** [comment]
#14706 got merged first...need a rebase here.

**Comment by yifan-gu** [comment]
Rebased

**Comment by yujuhong** [comment]
Kicked off shippable again. Hope this time it'd pass

**Comment by yujuhong** [comment]
Oops...need to rebase again

**Comment by yifan-gu** [comment]
rebased.

**Pull request merged**